frontend/sw360-portlet/src/main/webapp/js/modules/codeScoop.js

```diff
@@ -0,0 +1,966 @@
+/*
+ * Copyright Siemens AG, 2017.
```

Comment:
`Copyright Siemens AG, 2017. ` is correct?

Reply:
@maxhbr which copyright I should put here?

Reply:
I will clarify

Reply:
@mcjaeger Michael, do you have update for this issue? can i use such copyright?

Reply:
for entirely new files like this please use the copyright from your organisation.